styles.css

```diff
@@ -0,0 +1,1240 @@
+* {
+    box-sizing: border-box;
+    margin: 0%;
+    padding: 0%;
+
+}
+
+#layout {
+    height: 100vh;
+    display: grid;
+    grid-template:
+
+        "side    left-header   left-header     main-header     right-header   side2"50px "side    list-logo     list-logo    middle-main     main-right     side2"15% "side    list-logo    list-logo    main            main-right     side2"1fr "side     tweak         tweak    main              main-right    side2"75px "side     bottom-left  bottom-left     main            bottom-right  side2"70px/ 1fr 0.8fr 1.3fr 6fr 3fr 0.8fr;
+
+}
+
+.side {
+    grid-area: side;
+    background-color: white;
+    ;
+}
+
+.left-header {
+    grid-area: left-header;
+    margin-right: 70%;
+    margin-left: 10%;
+    margin-top: 5%;
+
+}
+
+.left-header img {
+    object-fit: contain;
+    height: 100%;
+    width: 100%;
+    border-radius: 60%;
+}
+
+.list-logo {
+    grid-area: list-logo;
+    display: grid;
+    grid-template-columns: repeat(2, auto);
+    margin-top: 4mm;
+    margin-left: 4mm;
+
+}
+
+.list-logo ul {
+    list-style: none;
+    line-height: 60px;
+}
+
+.list-logo img {
+    width: 25px;
+
+}
+
+.list-logo span {
+    font-weight: 550;
+    font-size: large;
+    font-family: Arial, Helvetica, sans-serif;
+
+}
+
+#home {
+    width: 115px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+}
+
+#explore {
+
+    width: 130px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+
+}
+
+#notifications {
+
+    width: 180px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#messages {
+
+    width: 155px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#bookmarks {
+
+    width: 165px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#lists {
+
+    width: 110px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#profile {
+
+    width: 125px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+#more11 {
+
+    width: 110px;
+    padding-left: 4mm;
+    border-radius: 4cm;
+    margin-top: 2mm;
+}
+
+
+#home:hover {
+    background-color: rgb(231 231 232);
```

Comment:
Suggested improvement: It looks like you're using this same colour a lot. You could use CSS Vars to create a variable, and reuse it elsewhere in your styles. You can see an example of doing this here: https://www.w3schools.com/css/css3_variables.asp